src/IonTests.ts

```diff
@@ -25,3 +29,55 @@ export {BinaryWriter} from "./IonBinaryWriter";
 export {ParserBinaryRaw} from "./IonParserBinaryRaw";
 export {BinarySpan} from "./IonSpan";
 export {Decimal} from "./IonDecimal";
+
+// Generates a mocha-friendly string representation of the provided value that can be used in test names
+export function valueName(value: any): string {
```

Comment:
This code was moved from `test/dom/Value.ts` so it could be referenced from multiple test files.

Reply:
Some investigation determined that the issue stemmed from `tsconfig.json` setting the project's `rootDir` to `src` rather than the actual root, which caused `test` to be considered a non-source directory. Setting `rootDir` to `.` (the same directory as `tsconfig.json` itself) lifted this restriction; it does not appear to have affected anything else.

I've created `test/exampleValues.ts` to house the functionality I had previously placed in `src/IonTests.ts`.

> I'd like to see `src/IonTests.ts` removed at some point

Now that we have a path forward, I've opened #582 to track this work.

Reply:
Thank you for taking a closer look.  I wonder if changing `rootDir` changes the paths produced within the `dist` folder...?

Reply:
It doesn't appear to. That behavior is set by `outDir`, which is not specified relative to `rootDir`.